### PR TITLE
Gena Monroe 

### PR DESCRIPTION
**Pull request by genarmonroe**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [x] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [ ] The webpage links to another page (multi-page website): 10 points
- [x] The company/orgnanization name is lined up with the navigation menu at the top of the page: 10 points
- [x] The title is centered in front of a background image: 10 points
- [x] The body images are sized and formatted according to the wire frame: 10 points
- [x] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [x] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [x] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [x] Responsive navbar: 20 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [ ] Matched layout of original site: 20 points
- [x] Matched font of original site: 10 points
- [ ] Properly commented code: 10 points
- [ ] Properly indented code: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [ ] Responive: 20 points
- [ ] Evidence of Effective Use of Design Principles: 20 points
- [ ] Links to Resume: 20 points
- [ ] Links to at least 3 Projects: 20 points
- [ ] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

File: 01week/bio/index.html [hunk 1]
```diff
@@ -19,12 +19,11 @@
     <br>
     <p><b>I'm a student at Austin Coding Academy and a fan of chickens... the pet that makes you breakfast.</b></p>
```
Comment (libersolis-dev) [inline 01week/bio/index.html]: Great job breaking up this content. I love how you are already using ids to target your content. You also are properly organizing your code with the use of headings and paragraphs. Everything appears to make logical, structural sense and is easy to read! 

This is purely a stylistic preference, and one that you will certainly find different opinions on, but you may want to consider using the `<strong></strong>` tag instead  of `<b></b>`. 
If you think about things semantically/accessibly strong could be understood, even in the absence of visible styling. Someone with a screen reader will be able to understand strong in relation to the document vs bold. 

As we get more into css you can further separate the presentational aspect of your document from the organizational aspect of your document by creating a class and using the 'font-weight' property to add bolder/lighter fonts to your document.

Below are a couple of articles on the topics discussed in the comments. Awesome job!

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong
https://css-tricks.com/almanac/properties/f/font-weight/

Comment (genarmonroe) [reply 01week/bio/index.html]: manny- thanks so much for your review, positive feedback, and suggestion! i will check out the links, i need allll the information i can get. thanks again! : )

File: 01week/bio/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <p id="name">S N I P P E T &nbsp B I O | G e n a &nbsp M o n r o e</p>
+    <a href="mailto:genarmonroe@gmail.com">
+    <p id="email">genarmonroe@gmail.com</p>
+    </a>
+    </div>
+  <div class="left"></div>
+  <div class="right">
+<!-Begin blabber for snippet content>
+    <br>
+      <img src="http://i.imgur.com/yPgNkAb.jpg?1" height="150" width="200" border="0" alt="The girls checking out a camera">
```
Comment (mymonstermaggie) [inline 01week/bio/index.html]: looks good to me. love the chickens!

File: 01week/bio/css/style.css [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+<link rel="stylesheet" href=".css/style.css/>
```
Comment (stefanbemelmans) [inline 01week/bio/css/style.css]: I think you need to lose the opening dot

File: 01week/bio/index.html (hunk 2 above)
Comment (genarmonroe) [reply 01week/bio/index.html]: thank you for your review!

**Pull request closed without merging**